tenable/ot/graphql/query.py

```diff
@@ -0,0 +1,423 @@
+      lastUpdate
+      location
+      macs {
+        nodes
+      }
+      model
+      name
+      os
+      osDetails {
+        name
+        architecture
+        hotFixes {
+          nodes {
+            name
+            installDate
+            description
+          }
+        }
+        version
+      }
+      purdueLevel
+      runStatus
+      runStatusTime
+      serial
+      slot
+      superType
+      type
+      vendor
+      risk {
+        unresolvedEvents
+        totalRisk
+      }
+      ips {
+        nodes
+      }
+      macs {
+        nodes
+      }
+      backplane {
+        id
+        name
+        size
+      }
+      segments {
+        nodes {
+          id
+          name
+          type
+          key
+          systemName
+          vlan
+          description
+          assetType
+          subnet
+        }
+      }
+      revisions {
+        nodes {
+          id
+          firstSeen
+          lastSeen
+          ordinal
+          isBase
+        }
+      }
+      plugins {
+        nodes {
+          id
+          name
+          source
+          family
+          severity
+          vprScore
+          comment
+          owner
+          totalAffectedAssets
+        }
+      }
+    }
+  }
+}
+"""
+PLUGINS_QUERY = """
+query plugins(
+  $filter: PluginExpressionsParams
+  $search: String
+  $sort: [PluginSortParams!]!
+  $limit: Int
+  $startAt: String
+) {
+  plugins(
+    filter: $filter
+    search: $search
+    sort: $sort
+    first: $limit
+    after: $startAt
+  ) {
+    pageInfo {
+      endCursor
+    }
+    nodes {
+      id
+      name
+      source
+      family
+      severity
+      vprScore
+      comment
+      owner
+      totalAffectedAssets
+      affectedAssets {
+        nodes {
+          id
+          name
+        }
+      }
+    }
+  }
+}
+"""
+PLUGINS_DETAILS_QUERY = """
+query plugins(
+  $filter: PluginExpressionsParams
+  $search: String
+  $sort: [PluginSortParams!]!
+  $limit: Int
+  $startAt: String
+) {
+  plugins(
+    filter: $filter
+    search: $search
+    sort: $sort
+    first: $limit
+    after: $startAt
+  ) {
+    pageInfo {
+      endCursor
+    }
+    nodes {
+      id
+      name
+      source
+      family
+      severity
+      vprScore
+      comment
+      owner
+      totalAffectedAssets
+      details {
+        id
+        name
+        source
+        family
+        description
+        solution
+        seeAlso
+        pluginType
+        pluginPubDate
+        pluginModDate
+        vulnPubDate
+        vulnModDate
+        refs {
+          name
+          value
+          url
+        }
+        cpe
+        cvssVector
+        cvssV3Vector
+        cvssBaseScore
+        cvssV3BaseScore
+        cvssV3TemporalScore
+        cvssTemporalScore
+        cvssV3TemporalVector
+        cvssImpactScore
+      }
+      affectedAssets {
+        nodes {
+          id
+          name
+        }
+      }
+    }
+  }
+}
+"""
+EVENTS_QUERY = """
```

Comment:
2 blank lines.  comment detailing what this constant is

Reply:
fixed